cli_tools_tests/e2e/gce_ovf_export/scripts/ovf_import_test_ubuntu_3_disks.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests to validate if Ubuntu OVF with 3 disks was imported successfully.
+
+
+# To disable checking for the OS config agent, add an instance metadata
```

Comment:
Thoughts on referencing the other script? (Rather than duplicating it)

cli_tools_tests/e2e/gce_ovf_import/scripts/ovf_import_test_ubuntu_3_disks.sh

Reply:
changed